lua/bufferline/render.lua

```diff
@@ -221,32 +197,32 @@ local function render(update_names)
   local items = {}
 
   local current_buffer_index = nil
-  local current_position = 0
+  local current_buffer_position = 0
   for i, buffer_number in ipairs(buffer_numbers) do
 
     local buffer_data = state.get_buffer_data(buffer_number)
     local buffer_name = buffer_data.name or '[no name]'
 
     buffer_data.real_width    = Layout.calculate_width(buffer_name, layout.base_width, layout.padding_width)
-    buffer_data.real_position = current_position
+    buffer_data.real_position = current_buffer_position
 
     local activity = Buffer.get_activity(buffer_number)
     local is_inactive = activity == 0
-    local is_visible = activity == 1
+    -- local is_visible = activity == 1
```

Comment:
`is_visible` is unused; I commented it just to show what `activity == 1` represents.